recipes/symmetrica/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "2.0" %}
+{% set name = "symmetrica" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.algorithm.uni-bayreuth.de/en/research/SYMMETRICA/SYM2_0_tar.gz
+  sha256: bf52788dedc14c482e89f5e7efe8c60864a633314ddd446dd4602d5fdaca0ee2
+
+  # patches are not added here, because the files in the tarball have permission 400 and
```

Comment:
Oh. It is the tarball and not the patches. I read it wrong. Not sure what we can do then...